gemrb/GUIScripts/Spellbook.py

```diff
@@ -97,6 +97,16 @@ def GetKnownSpells(actor, BookType):
 
 	return knownSpells
 
+def GetKnownSpellsDescription(actor, BookType):
+        """ Gets text to display in the chargen spell listing. """
+	info = ""
+        Spells = GetKnownSpells(actor, BookType)
+        # reverse spells order grouped by SpellLevel
+        Spells.sort(lambda fst, snd: -1 if fst['SpellLevel'] == snd['SpellLevel'] else 0)
+        for Spell in Spells:
+		info += GemRB.GetString (Spell['SpellName']) + "\n"
+	return info
+
```

Comment:
```suggestion
def GetKnownSpellsDescription(actor, BookType):
	""" Prepares known and memorized spell listing for chargen. """

	info = ""
	Spells = GetKnownSpells(actor, BookType)
	# reverse spells order grouped by SpellLevel
	Spells.sort(lambda fst, snd: -1 if fst['SpellLevel'] == snd['SpellLevel'] else 0)
	for Spell in Spells:
		info += GemRB.GetString (Spell['SpellName']) + "\n"
	return info

```
Trying to fix it it for you via web, so I can merge sooner.